### PR TITLE
Add Item

### LINKED ISSUE #47: Add item
Elias will finish this... he knows the way

### PR DESCRIPTION
**Pull request by Eliascm17**
helpppppppp 😬 😎

closes #47

### CODE REVIEW

File: google-services.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (ynigoreyes) [inline google-services.json:1]: I believe this is for non-managed mobile applications. Check [this](https://medium.com/@denisceric94/expo-and-firebase-authentication-email-password-facebook-and-google-681a511c61b6) out

File: src/screens/Athena.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import Onboarding from './onboarding/Onboarding'
+import Navbar from './navbar/Navbar'
+
+const Athena = () => {
+
+    const { state } = useUserContext()
+
+    return (
+        <>
+            { state.user ? <Navbar /> : <Onboarding />}
```
Comment (ynigoreyes) [inline src/screens/Athena.jsx:12]: Careful with this line. I believe state.user is always defined because of the default value of state.user is an empty object literal, {}. I would check for state.user.id instead

**Review by ynigoreyes — COMMENTED** [review]
I added a guide for you to look at. Hope it helps! Looks like I was wrong about using the web API. You do gotta use something expo-specific. I would definitely take all the shortcuts that you can and using third-party libraries and whatnot. It's just authentication, we need it to be quick, simple, and not take up too much time in the development process.

**Comment by ynigoreyes** [comment]
I can take a look at this after I'm done with the home screen. You should work on a different part of the app and just come back to this later. I feel like there are still more important parts that need to get done

**Comment by Eliascm17** [comment]
Sounds good, I'm gonna work on add item

File: GoogleService-Info.plist [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ynigoreyes) [inline GoogleService-Info.plist:1]: Was this file still necessary?

**Review by ynigoreyes — APPROVED** [review]
